cmd/agent/app/integrations.go

```diff
@@ -22,6 +22,7 @@ import (
 	"github.com/DataDog/datadog-agent/pkg/config"
 	"github.com/DataDog/datadog-agent/pkg/util/executable"
 
+	"github.com/coreos/go-semver/semver"
```

Comment:
since this package is now a direct dependency of the project, please add it to `Gopkg.toml`, and set a reasonable version constraint on it

Reply:
It's being pulled in as a sub-dependency by some other package already, so that's probably a reasonable place to set the pin in `Gopkg.toml` as Olivier suggests above.:
```
[[projects]]
  digest = "1:3c3f68ebab415344aef64363d23471e953a4715645115604aaf57923ae904f5e"
  name = "github.com/coreos/go-semver"
  packages = ["semver"]
  pruneopts = ""
  revision = "8ab6407b697782a06568d4b7f1db25550ec2e4c6"
  version = "v0.2.0"
```